runtime/extensions/evm/src/executor/util.rs

```diff
@@ -0,0 +1,181 @@
+// Copyright 2019-2020 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+use evm::Opcode;
+extern crate alloc;
+use alloc::string::ToString;
+use sp_std::vec::Vec;
+
+pub fn opcodes(opcode: Opcode) -> Vec<u8> {
```

Comment:
Is this our own custom mapping or a published thing? Can we import it from somewhere?

Reply:
I search around at some point and couldn't find any. Now the EVM uses a lookup table sand we cannot import it from there, so I ended up just writting it myself.

I double checked that is correctly mapped, but all eyes on this are welcome!